tools/lint-hotkeys

```diff
@@ -67,16 +67,16 @@ def lint_hotkeys_file() -> None:
             print(
                 f"Duplicate key combination for keys {duplicate_keys} for category ({HELP_CATEGORIES[action]}) detected"
             )
-            error_flag = 1
-    if error_flag == 1:
+            error_flag = True
+    if error_flag:
         print(f"Rerun this command after resolving errors in config/{KEYS_FILE_NAME}")
     else:
         print("No hotkeys linting errors")
         if not output_file_matches_string(hotkeys_file_string):
             print(
                 f"Run './tools/{SCRIPT_NAME} --fix' to update {OUTPUT_FILE_NAME} file"
             )
-            error_flag = 1
+            error_flag = True
     sys.exit(error_flag)
```

Comment:
While this likely works, an explicit conversion here would be good, as per https://github.com/zulip/zulip-terminal/pull/1518#discussion_r1657703081